native/cocos/audio/android/audio_utils/include/audio_utils/primitives.h

```diff
@@ -19,9 +19,13 @@
 
 #include <stdint.h>
 #include <stdlib.h>
+#if CC_PLATFORM == CC_PLATFORM_ANDROID
 #include <sys/cdefs.h>
+#endif
 
-__BEGIN_DECLS
+#ifdef	__cplusplus
+extern "C" {
```

Comment:
same as above

Reply:
It's better to add it, because it includes both c and C++ files, and it should be safer to add this